src/Stripe.net/Infrastructure/Requestor.cs

```diff
@@ -179,6 +179,11 @@ private static void ApplyMultiPartFileToRequest(HttpRequestMessage requestMessag
         {
             requestMessage.Headers.ExpectContinue = true;
 
+            if (string.IsNullOrEmpty(fileName))
+            {
+                fileName = "blob";
```

Comment:
should we be more explicit to indicate it's a hardcoded value? blob has some meaning for me as a type (like in a database) and so it felt weird the first 2 times I read the PR.

No big deal obviously but wanted to flag

Reply:
"blob" is used with more or less the same meaning as in a database: it indicates that what follows is a binary large object. It's also the convention we already use in the [Node](https://github.com/stripe/stripe-node/blob/a7382226965ee399ce6b5b9fd162c5af74c8131d/lib/MultipartDataGenerator.js#L30) and [Python](https://github.com/stripe/stripe-python/blob/f948b8b95b6df5b57c7444a05d6c83c8c5e6a0ac/stripe/multipart_data_generator.py#L24) libraries.

Reply:
gotcha, works for me if we do this in the other libs!